.github/workflows/test.yml

```diff
@@ -50,3 +50,9 @@ jobs:
       - name: Run the tests
         run: poetry run pytest tests/integration
         working-directory: cli
+        env:
+          # Credentials used for the test_infra/ tests
+          SCW_DEFAULT_ORGANIZATION_ID: ${{ secrets.SCW_DEFAULT_ORGANIZATION_ID }}
+          SCW_DEFAULT_PROJECT_ID: ${{ secrets.SCW_DEFAULT_PROJECT_ID }}
+          SCW_SECRET_KEY: ${{ secrets.SCW_SECRET_KEY }}
+          SCW_ACCESS_KEY: ${{ secrets.SCW_ACCESS_KEY }}
```

Comment:
This shouldn't be needed for running the tests locally against the docker-compose environment. 

I think we need to exclude the Cockpit and Domains tests when we're _not_ running with `DEPLOY_ENV=scw`